### ISSUE TITLE
[native-image] Adding resources from the external jar

### ISSUE THREAD

**Issue by PatrykGala** [oracle/graal#1062]
In Java project, I've added the dependency to DynamoDB SDK in Gradle file:
 `compile group: 'com.amazonaws', name: 'aws-java-sdk-dynamodb', version: '1.11.515'`

After build the native image and run application I have following issue:
`Message: Error while loading partitions file from com/amazonaws/partitions/endpoints.json`

I've added a configuration to native-image (below) but this doesn't work...
`-H:IncludeResources=".*endpoints.*json.*"`

The file exists in the jar and this is the source: [endpoints.json](https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/resources/com/amazonaws/partitions/endpoints.json)

How should I configure native-image to attach resources from others jars?

**Comment by tomerz90** [comment]
@PatrykGala HAve you tried using an exact name? `-H:IncludeResources="endpoints.json"` ? Worked for me in another case, also where did you find docs for this flag? I couldnt find any and im trying to get it to work to include an entire directory

**Comment by cstancu** [comment]
Can you try `-H:IncludeResources='.*/endpoints.json$'`? Please see the [documentation](https://github.com/oracle/graal/blob/master/substratevm/RESOURCES.md#accessing-resources-in-substrate-vm-images) for more examples.

**Comment by jordanms** [comment]
I'm using the latest version available today (22.3.0) and I have the same issue. I have tried everything and it doesn't work.

I'm using `dss-asic-xades` library. I also have my own resources (`/src/main/resources`) and the library resources to add.

To reproduce the problem just create a simple project that uses it:

**pom.xml**
```
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
...
<dependencies>
    <dependency>
      <groupId>org.graalvm.nativeimage</groupId>
      <artifactId>svm</artifactId>
      <version>22.3.0</version>
    </dependency>
    <dependency>
      <groupId>eu.europa.ec.joinup.sd-dss</groupId>
      <artifactId>dss-utils-apache-commons</artifactId>
    </dependency>
    <dependency>
      <groupId>eu.europa.ec.joinup.sd-dss</groupId>
      <artifactId>dss-asic-xades</artifactId>
    </dependency>
...
  </dependencies>

  <build>
    <finalName>asice-xades-dss</finalName>
    <plugins>
      <plugin>
        <groupId>org.graalvm.nativeimage</groupId>
        <artifactId>native-image-maven-plugin</artifactId>
        <version>21.2.0</version>
        <executions>
          <execution>
            <goals>
              <goal>native-image</goal>
            </goals>
            <phase>package</phase>
          </execution>
        </executions>
        <configuration>
          <buildArgs>--shared -H:Name=asice-xades-dss -H:ResourceConfigurationFiles=${basedir}/src/main/resources/resource-config.json</buildArgs>
        </configuration>
      </plugin>
    </plugins>
  </build>

  <repositories>
    <repository>
      <id>cefdigital</id>
      <name>cefdigital</name>
      <url>https://ec.europa.eu/digital-building-blocks/artifact/content/repositories/esignaturedss/</url>
    </repository>
  </repositories>

  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>eu.europa.ec.joinup.sd-dss</groupId>
        <artifactId>dss-bom</artifactId>
        <version>5.11</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
</project>
```
**Entre class**
```
  @CEntryPoint(name = "start_signing")
  public static int startSigning(IsolateThread thread) {
    new CommonCertificateVerifier();
    return 0;
  }
```
**resource-config.json** (tried everything and nothing works)
```
// 1. my local resource is included, library resource is not
{
  "resources": {
    "includes": [
      {"pattern": ".*xml$"},
      {"pattern": ".*/constraint.*xml$"},
      {"pattern": ".*/.*xsd$"}
    ]
  }
}

// 2. my local resource is included, library resource is not
{
  "resources": {
    "includes": [
      {"pattern": ".*xml$"},
      {"pattern": ".*/.*xsd$"}
    ]
  }
}

// 3. my local resource is not included, library resource is not.
{
  "resources": {
    "includes": [
      {"pattern": ".*/.*xml$"},
      {"pattern": ".*/.*xsd$"}
    ]
  }
}
```

Error message
```
eu.europa.esig.dss.model.DSSException: Unable to instantiate default RevocationDataVerifier. Reason : null
    at eu.europa.esig.dss.validation.RevocationDataVerifier.createDefaultRevocationDataVerifier(RevocationDataVerifier.java:109)
    at eu.europa.esig.dss.validation.CommonCertificateVerifier.<init>(CommonCertificateVerifier.java:86)
    at eu.europa.esig.dss.validation.CommonCertificateVerifier.<init>(CommonCertificateVerifier.java:176)
    at my.package.ServiceBiding.startSigning(ServiceBiding.java:31)
```

**Comment by olpaw** [comment]
Feel free to reopen if you are still not able to include the resources shown above.
